cmds/core/backoff/main_test.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2021 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"bytes"
+	"os/exec"
+	"regexp"
+	"testing"
+
+	"github.com/u-root/u-root/pkg/testutil"
+)
+
+type test struct {
+	args   []string
+	o      string
+	e      string
+	exitok bool
+}
+
+func run(c *exec.Cmd) (string, string, error) {
+	var o, e bytes.Buffer
+	c.Stdout, c.Stderr = &o, &e
+	err := c.Run()
+	return o.String(), e.String(), err
+}
+
+// TestOK for now just runs a simple successful test with 0 args or more than one arg.
+func TestOK(t *testing.T) {
+	var tests = []test{
+		{args: []string{}, o: "", exitok: true},
+		{args: []string{"date"}, o: ".*", exitok: true},
+		{args: []string{"-t", "wh", "date"}, o: ".*", e: ".*invalid.*duration.*wh", exitok: false},
+		{args: []string{"echo", "hi"}, o: ".*hi", exitok: true},
+	}
+
+	for _, v := range tests {
+		c := testutil.Command(t, v.args...)
+		o, e, err := run(c)
```

Comment:
it's hard to distinguish between e and err reading this, spell them out?

stdout, stderr, err :=

Reply:
done